src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.input {
+  display: block;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  outline: none;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-weight: bold;
+  font-size: 16px;
+  color: #3d4e61;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px;
+  margin: 20px 0;
+  padding: 20px 62px;
+  height: 70px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background-image: url(images/Search.svg), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat, no-repeat;
```

Comment:
for what  two times no-repeat?

Reply:
I seen these in exemple, when have learned multiple backgrounds:
![image](https://user-images.githubusercontent.com/123530320/216900651-67d4fabb-0ee7-4505-8f79-64b610b62341.png)